app/models/user.rb

```diff
@@ -1,6 +1,5 @@
 class User < ApplicationRecord
-  has_secure_password
```

Comment:
why you delete

Reply:
because i have 2 has_secure_password, so i remove one of it.